File: 09/main.cpp
#include <iostream>
#include <vector>

using namespace std;
class Solution {
public:
	int jumpFloorII(int number) {
		if (number <= 0)
			return 0;
		else if (number == 1)
			return 1;
		else
			return 2 * jumpFloorII(number - 1);
	}
};

int main(){
	Solution so;
	int n;
	cin >> n;
	int f = so.jumpFloorII(n);
	cout << "Total method: " << f << endl;

	getchar(); getchar();
	return 0;
}

File: 09/main.py
# -*- coding:utf-8 -*-
class Solution:
    def __init__(self):
        pass
    def jumpFloorII(self, number):
        if number<=0:
            return 0;
        elif number==1:
            return 1;
        else:
            return 2*self.jumpFloorII(number-1)
    
so=Solution()
x=int(raw_input())
f=so.jumpFloorII(x)
print "Total method:{}".format(f)